src/dispatch/individual/models.py

```diff
@@ -63,12 +63,21 @@ class IndividualContact(Base, ContactMixin, ProjectMixin):
 
 
 class IndividualContactBase(ContactBase):
-    weblink: Optional[str] = Field(None, nullable=True)
+    weblink: Union[HttpUrl, None, str] = Field(None, nullable=True)
     mobile_phone: Optional[str] = Field(None, nullable=True)
     office_phone: Optional[str] = Field(None, nullable=True)
     title: Optional[str] = Field(None, nullable=True)
     external_id: Optional[str] = Field(None, nullable=True)
 
+    @validator("weblink")
+    def weblink_validator(cls, v):
+        if v is None:
+            return v
+        if not isinstance(v, HttpUrl) and len(v) > 0:
+            raise ValueError(
+                "weblink is not an empty string and not a valid weblink")
+        return v
```

Comment:
this is a small nit, but I prefer something that is more explicit and a raise if not.
```suggestion
        if v is None or v == "" or isinstance(v, HttpUrl):
            return v

        raise ValueError("weblink is not an empty string or a valid weblink")
```

Reply:
I forget which order these need to be in so you could reverse the last two to `isinstance(v, HttpUrl) or v == ""`

Reply:
I reversed the last two. Thanks for helping clean up the logic!